src/test/java/tasks/DeadlineTest.java

```diff
@@ -0,0 +1,31 @@
+package tasks;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeadlineTest {
+    @Test
+    public void createDeadline_unmarked_success() throws Exception {
+        // toString
+        Deadline testDeadline1 = new Deadline("test description", new DateAndTime("2024-08-30"));
```

Comment:
Could you replace these names that use numbers with more descriptive names that elaborate what type of edge case is being tested?